Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -35,6 +35,13 @@ public class NetworkServer : MonoBehaviour
         public NetworkConnectionEvent Connected = new NetworkConnectionEvent();
         public NetworkConnectionEvent Authenticated = new NetworkConnectionEvent();
         public NetworkConnectionEvent Disconnected = new NetworkConnectionEvent();
+
+        /// <summary>
+        /// This is invoked when a server is started - including when a host is started.
+        /// <para>StartServer has multiple signatures, but they all cause this hook to be called.</para>
+        /// </summary>
+        public UnityEvent OnStartServer = new UnityEvent();
```

Comment:
Rename to "Started". for consistency with the other events.

We are already in the NetworkServer class,  so the word "Server" is redundant.

Also move above the other events,  this is the first one that gets fired

Reply:
fixed